src.py

```diff
@@ -0,0 +1,9 @@
+import pathlib
+import re
+
+PACKAGE = "httpx"
+
+
+def get_version(package=PACKAGE):
+    version = pathlib.Path(package, "__version__.py").read_text()
+    return re.search("__version__ = ['\"]([^'\"]+)['\"]", version).group(1)
```

Comment:
Should we move this part into `httpx/httpx/__version__.py`

Reply:
No because you can't import httpx until after setup

Reply:
Honestly this one edge-case has me not wanting to merge this. :(

Reply:
`setup.py` already does not import `http.__version__.__version__` because of this limitation. It's not an edge case

Reply:
I meant having this file to accomplish only this one task.